migration/migrator/data/course_tables.sql

```diff
@@ -13,20 +13,6 @@ SET xmloption = content;
 SET client_min_messages = warning;
```

Comment:
These changes were generated from the migrator `dump` function?

Reply:
yes.  I haven't modified any of the dumped files except by running the migrator as far as I am aware.  I've seen similar changes in other PRs I've reviewed.  I don't know enough SQL to know what purpose these serve or what may be causing the issue.

Reply:
The changes are fine. It's just shuffling around defaults with the newest versions of postgres. The stuff removed was mostly just compatibility with much older (e.g. postgres 8) versions we don't need to worry about.